src/main/resources/config/en.yml

```diff
@@ -8,6 +8,11 @@ ConfigVersion: ${version}
 # You must restart your server after changing this option
 BotToken: "BOTTOKEN"
 
+ProxyHost: "https://example.com"  
```

Comment:
```suggestion
# The proxy to be used for Discord
# Leave this alone if you don't understand what it does
ProxyHost: "https://example.com"  
```
Needs a comment explaining to people what this is used for. Copy this english translation across all the config translations you can't translate.